webpack/assets/javascripts/react_app/components/users/JwtTokens/JwtTokens.js

```diff
@@ -0,0 +1,64 @@
+import React, { Fragment } from 'react';
+import { Button } from '@patternfly/react-core';
+import { KeyIcon } from '@patternfly/react-icons';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { openConfirmModal } from '../../ConfirmModal';
+import { APIActions } from '../../../redux/API';
+import { translate as __ } from '../../../common/I18n';
+
+const JwtTokens = ({ userId }) => {
+  const dispatch = useDispatch();
+  return (
+    <Fragment>
+      <table className="table table-bordered table-striped table-hover table-fixed">
+        <tbody>
+          <tr>
+            <td className="blank-slate-pf">
+              <div className="blank-slate-pf-icon">
+                <KeyIcon type="fa" name="key" color="#9c9c9c" />
+              </div>
+              <h1>{__('JWT Tokens')}</h1>
+              <p>
+                {__(
+                  'By invalidating your JSON Web Tokens (JWTs), you will no longer be able to register hosts by using your existing JWTs.'
+                )}
+              </p>
+              <Button
+                ouiaId="invalidate-jwt-token-button"
+                variant="primary"
+                isSmall
+                onClick={() =>
+                  dispatch(
+                    openConfirmModal({
+                      isWarning: true,
+                      title: __('Invalidate tokens for self?'),
+                      confirmButtonText: __('Confirm'),
+                      onConfirm: () =>
+                        dispatch(
+                          APIActions.patch({
+                            url: `/users/${userId}/invalidate_jwt`,
+                            key: `INVALIDATE-JWT`,
+                            successToast: () =>
```

Comment:
What if the call is not successful?

Reply:
That will not happen, Like we discussed in the first story PR, there will be no case when invalidation won't work. I can add a safety failed toast, what do you think?

Reply:
What if user was logged out meanwhile? The Rails will respond with 501

Reply:
we are sending `:ok` status at all times, [ref](https://github.com/theforeman/foreman/pull/10405/commits/a25111626f9da7e74dd250e6c0706679a7f2828e#diff-cfdccd0a9d5df5a43aaad2a35d36ebbe187c52ad5fdc9846fa189d04537adb6eR106)
I think I can add a safety error toast too, what do u think?

Reply:
> we are sending :ok status at all time

That's not true. You do in>your< controller, but what if there is a problem on the network, DNS, or somewhere else? How do you handle that error?

Reply:
updated